web/src/app/schedules/ScheduleOverrideForm.js

```diff
@@ -38,163 +39,162 @@ const query = gql`
-                fullWidth
-                component={UserSelect}
-                required
-                name='addUserID'
-                label='User to Add'
-              />
-            </Grid>
-          )}
+            Start and end time shown in {zone === 'local' ? 'local time' : zone}
+            .
+          </Typography>
+        </Grid>
+        <Grid item xs={12} sm={12} md={6}>
+          {/* Purposefully leaving out of form, as it's only used for converting display times. */}
+          <ScheduleTZFilter
+            label={tz => `Configure in ${tz}`}
+            scheduleID={scheduleID}
+          />
+        </Grid>
+        {remove && (
           <Grid item xs={12}>
             <FormField
               fullWidth
-              component={DateTimePicker}
-              mapValue={value => DateTime.fromISO(value, { zone })}
-              mapOnChangeValue={value => value.toISO()}
-              showTodayButton
+              component={UserSelect}
+              name='removeUserID'
+              label={add && remove ? 'User to be Replaced' : 'User to Remove'}
               required
-              name='start'
-              leftArrowIcon={<ChevronLeft />}
-              rightArrowIcon={<ChevronRight />}
-              InputProps={{
-                endAdornment: (
-                  <InputAdornment position='end'>
-                    <IconButton>
-                      <DateRange />
-                    </IconButton>
-                  </InputAdornment>
-                ),
-              }}
             />
           </Grid>
+        )}
+        {add && (
           <Grid item xs={12}>
             <FormField
               fullWidth
-              component={DateTimePicker}
-              mapValue={value => DateTime.fromISO(value, { zone })}
-              mapOnChangeValue={value => value.toISO()}
-              showTodayButton
-              name='end'
+              component={UserSelect}
               required
-              leftArrowIcon={<ChevronLeft />}
-              rightArrowIcon={<ChevronRight />}
-              InputProps={{
-                endAdornment: (
-                  <InputAdornment position='end'>
-                    <IconButton>
-                      <DateRange />
-                    </IconButton>
-                  </InputAdornment>
-                ),
-              }}
+              name='addUserID'
+              label='User to Add'
             />
           </Grid>
-          {userError && <DialogContentError error={userError.message} />}
+        )}
+        <Grid item xs={12}>
+          <FormField
+            fullWidth
+            component={DateTimePicker}
+            mapValue={value => DateTime.fromISO(value, { zone })}
+            mapOnChangeValue={value => value.toISO()}
+            showTodayButton
+            required
+            name='start'
+            leftArrowIcon={<ChevronLeft />}
+            rightArrowIcon={<ChevronRight />}
+            InputProps={{
```

Comment:
Also a nit, but looks like leftArrowIcon and rightArrowIcon are defaulted to the left and right chevron unless otherwise set.

Reply:
Sounds good. I didn't actually change any of the JSX, just moved it all around. Looks like the package may have been updated with those defaults somewhere along the line. I'll go ahead and remove the icons